src/bootstrap.js

```diff
@@ -112,9 +123,7 @@ module.exports = (ceConfig, pnConfig) => {
             // we don't do auth on this one because it's assumed to be done with the /auth request below
             ChatEngine.global = new Chat(ChatEngine, ceConfig.globalChannel, false, true, 'global');
 
-            // create a new user that represents this client
-            ChatEngine.me = new Me(ChatEngine, pnConfig.uuid, authData);
-            ChatEngine.me.update(state);
+            ChatEngine.me = buildMe();
```

Comment:
Was there anything behind decision to make `Me` creation as separate function.
Bootstrap is already pretty big and there won't be any re-use of code, which create local user (`Me`) since it happens only once during chat engine instantiation. 
In amount of code it is 11 new lines against 4 (if instance assigned to `ChatEngine.me` directly and called required methods on it. 

I may be don't know this pattern, if so, tell me and I will approve since all other looks good now.

Reply:
It was a small attempt to begin splitting bootstrap.js into a bunch of small functions.

You're correct about code reuse, it would be better to do a complete refactor of this file in the future, suitable for individual unit tests,